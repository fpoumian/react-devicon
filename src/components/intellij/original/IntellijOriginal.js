import React from 'react'
import PropTypes from 'prop-types'
import SVGDeviconInline from '../../_base/SVGDeviconInline'

import iconSVG from './IntellijOriginal.svg'

/** IntellijOriginal */
function IntellijOriginal({ width, height, className }) {
  return (
    <SVGDeviconInline
      className={'IntellijOriginal' + ' ' + className}
      iconSVG={iconSVG}
      width={width}
      height={height}
    />
  )
}

IntellijOriginal.propTypes = {
  className: PropTypes.string,
  width: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
  height: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
}

export default IntellijOriginal
