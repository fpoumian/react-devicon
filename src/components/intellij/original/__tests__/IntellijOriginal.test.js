import React from 'react'
import renderer from 'react-test-renderer'
import IntellijOriginal from '../index.js'

it('renders correctly', () => {
  const tree = renderer.create(<IntellijOriginal />).toJSON()
  expect(tree).toMatchSnapshot()
})
