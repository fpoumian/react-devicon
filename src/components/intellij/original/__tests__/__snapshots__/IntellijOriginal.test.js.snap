// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="SVGInline IntellijOriginal undefined"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<svg class=\\"SVGInline-devicon IntellijOriginal-devicon undefined-devicon\\" id=\\"intelij\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 128 128\\"><defs><style>.cls-1{fill:#fa930d}.cls-2{fill:#136ba2}</style></defs><title>Artboard 14</title><g id=\\"original\\"><path class=\\"cls-1\\" d=\\"M5.55 64.73a17.08 17.08 0 1 0 17.08-17.09A17.11 17.11 0 0 0 5.55 64.73z\\"/><path class=\\"cls-2\\" d=\\"M107.48 4.73c-9.39 0-15 7.56-15 20.23v77.8c0 12.66 5.59 20.23 15 20.23s15-7.56 15-20.23V25c-.03-12.7-5.63-20.27-15-20.27zM81.25 6.38H19.88a14.92 14.92 0 0 0 0 29.81v.11h31.41v28.4a28.7 28.7 0 0 1-28.63 28.7h-.77l-.08.1c-.42 0-.85-.07-1.29-.07a14.94 14.94 0 1 0 0 29.87c.45 0 .9 0 1.34-.07v.05h.78a58.65 58.65 0 0 0 58.61-58.55v-.54-9.59-.11V6.38z\\"/></g></svg>",
    }
  }
/>
`;
